### PR TITLE
Add SGI and TGA preview support

### PR DESCRIPTION
**Pull request by skjnldsv**
SGI AND TGA are supported by imagemagick natively 

https://imagemagick.org/script/formats.php

Sample files: https://filesamples.com/formats/sgi https://filesamples.com/formats/tga

image

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -358,7 +358,8 @@ Simon Birnbach <simon@simon-birnbach.de>
 Simon Könnecke <simonkoennecke@gmail.com>
 Simon Whittaker <simon@swbh.net>
 Sjors van der Pluijm <sjors@desjors.nl> Sjors van der Pluijm <sjors@youngguns.nl>
-John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com> John Molakvoæ <skjnldsv@users.noreply.github.com>
+John Molakvoæ <skjnldsv@protonmail.com> John Molakvoæ <skjnldsv@users.noreply.github.com>
+John Molakvoæ <skjnldsv@protonmail.com> John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>
```
Comment (skjnldsv) [inline .mailmap:362]: I know it's not directly related, but it updated some license header here as well, I guess it's still not that big of a change. Fell free to tell me if that's really stupid :)

File: resources/config/mimetypealiases.dist.json [hunk 2]
```diff
@@ -104,6 +104,8 @@
 	"application/internet-shortcut": "link",
 	"application/km": "mindmap",
 	"application/x-freemind": "mindmap",
-	"application/vnd.xmind.workbook": "mindmap"
+	"application/vnd.xmind.workbook": "mindmap",
+	"image/targa": "image/tga",
+	"image/targa": "image/tga"
```
Comment (rullzer) [inline resources/config/mimetypealiases.dist.json:109]: duplicate

Comment (rullzer) [reply resources/config/mimetypealiases.dist.json:109]: Or rather you need the other format right?

**Review by rullzer — CHANGES_REQUESTED** [review]

File: resources/config/mimetypealiases.dist.json (hunk 2 above)
Comment (skjnldsv) [reply resources/config/mimetypealiases.dist.json:109]: Yes, it's a typo, I meant to alias image/targa to image/tga only

File: .mailmap (hunk 1 above)
Comment (nickvergessen) [reply .mailmap:362]: add it to https://github.com/nextcloud/server/blob/master/.mailmap to map all those git user+email combinations to one entry

**Review by rullzer — APPROVED** [review]
Fine by me

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**